modules/xmpp/strophe.jingle.js

```diff
@@ -207,7 +207,7 @@ class JingleConnectionPlugin extends ConnectionPlugin {
             sess.replaceTransport($(iq).find('>jingle'), () => {
                 const successTime = window.performance.now();
 
-                logger.info('(TIME) Transport replace success!', successTime);
+                logger.info('(TIME) Transport replace success:\n', successTime);
```

Comment:
Is this intentional or it was supposed to be `\t`?

Reply:
🤦‍♂ yes should be \t